xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexTradeServiceRaw.java

```diff
@@ -208,7 +245,12 @@ private BitfinexOrderStatusResponse sendLimitOrder(
               isHidden,
               isPostOnly,
               useRemaining);
-      response = bitfinex.replaceOrder(apiKey, payloadCreator, signatureCreator, request);
+      response =
+          decorateApiCall(
+                  () -> bitfinex.replaceOrder(apiKey, payloadCreator, signatureCreator, request))
+              .withRetry(retry("trade-limitOrder"))
```

Comment:
non-idempotent requests should not have retries on them, you don't want to mistakenly double send an order